src/gui/widgets/Knob.cpp

```diff
@@ -487,6 +545,14 @@ void Knob::changeEvent(QEvent * ev)
 		m_cache = QImage();
 		update();
 	}
+	else if (ev->type() == QEvent::FontChange)
```

Comment:
There's font change now?

Reply:
This was added for the case that a `Knob` is created and a font is set as the very last action. In that case the new size of the knob widget has to be recomputed to accommodate for the changed label size.

Reply:
Okk then